pluto-plugins/plugins/logger/lib/src/main/java/com/pluto/plugins/logger/internal/LogsViewModel.kt

```diff
@@ -1,15 +1,23 @@
 package com.pluto.plugins.logger.internal
 
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import com.pluto.plugins.logger.internal.persistence.LogDao
+import com.pluto.plugins.logger.internal.persistence.database.DatabaseManager
 import com.pluto.utilities.extensions.asFormattedDate
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 
-internal class LogsViewModel : ViewModel() {
+internal class LogsViewModel(application: Application) : AndroidViewModel(application) {
+
+    private val logDao: LogDao by lazy { DatabaseManager(application.applicationContext).db.exceptionDao() }
```

Comment:
instead of creating new instance of `logDao`, we can use `LogDBHandler` here

Reply:
done